packages/@ember/-internals/glimmer/lib/setup-registry.ts

```diff
@@ -39,9 +39,7 @@ export function setupApplicationRegistry(registry: Registry): void {
   registry.register(P`template:-root`, RootTemplate as any);
   registry.injection('renderer', 'rootTemplate', P`template:-root`);
 
-  registry.register('renderer:-dom', InteractiveRenderer);
-  registry.register('renderer:-inert', InertRenderer);
-
+  registry.register('renderer:-main', Renderer);
```

Comment:
```suggestion
  registry.register('renderer:-dom', Renderer);
```

https://emberobserver.com/code-search?codeQuery=renderer%3A-dom